### PR TITLE
Create ListCard component and add it to Storybook

### PR DESCRIPTION
**Pull request by SebastianStupp**
Storybook:                                                                                                                                     
 >https://loginicer-pi-listcard-yjelgvrw.herokuapp.com/storybook/index.html

### CODE REVIEW

File: client/src/components/ListCard.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+const ListCardContainer = styled.div`
+  display: flex;
+  background-color: ${(props) => props.theme.colors.primary};
+  width: 100%;
+  max-width: 500px;
+  height: 20vh;
+  max-height: 150px;
+  color: ${(props) => props.theme.colors.textprimary};
+  padding: 0px;
```
Comment (lmachens) [inline client/src/components/ListCard.js:15]: default is `0px
```suggestion

```

File: client/src/components/ListCard.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import Edit from '../assets/edit.svg';
+import Delete from '../assets/delete.svg';
+import { action } from '@storybook/addon-actions';
+
+const ListCardContainer = styled.div`
+  display: flex;
+  background-color: ${(props) => props.theme.colors.primary};
+  width: 100%;
+  max-width: 500px;
+  height: 20vh;
```
Comment (lmachens) [inline client/src/components/ListCard.js:12]: It's dangerous to set the container height related to the viewport. 
Instead, calculate the height based on the content set at a fixed height. You already limit the max-height.

File: client/src/components/ListCard.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import styled from '@emotion/styled';
+import Edit from '../assets/edit.svg';
+import Delete from '../assets/delete.svg';
+import { action } from '@storybook/addon-actions';
+
+const ListCardContainer = styled.div`
+  display: flex;
+  background-color: ${(props) => props.theme.colors.primary};
+  width: 100%;
+  max-width: 500px;
```
Comment (lmachens) [inline client/src/components/ListCard.js:11]: If you select `Pixel` or a device with a larger screen, you will face this issue:

image

File: client/src/components/ListCard.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+const ListSymbolContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-basis: 20%;
+  flex-direction: column;
+  justify-content: space-around;
+`;
+
+const ListSymbolImage = styled.img``;
```
Comment (lmachens) [inline client/src/components/ListCard.js:36]: There is no reason to create a styled component with empty styles.
Just use ``

File: client/src/components/ListCard.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+const ListExampleContent = ['ClientNameOne', 'ClientNameTwo'];
```
Comment (lmachens) [inline client/src/components/ListCard.js:38]:
```suggestion
const listExampleContent = ['ClientNameOne', 'ClientNameTwo'];
```

PascalCase is for React components

File: client/src/components/ListCard.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+          src={Delete}
+          alt="Delete Symbol"
+          onClick={action('click')}
+        ></ListSymbolImage>
+      </ListSymbolContainer>
+    </ListCardContainer>
+  ));
+};
+
+export default function DefaultListCard() {
```
Comment (lmachens) [inline client/src/components/ListCard.js:60]: The filename and default export should match.
`ListCard.js` should return a React component called `ListCard`.

File: client/src/components/ListCard.js (hunk 5 above)
Comment (lmachens) [reply client/src/components/ListCard.js:38]: Actually, this example should be in your story and not inside the component. 
Stories are great for test/example usage.

File: client/src/components/ListCard.js (hunk 2 above)
Comment (SebastianStupp) [reply client/src/components/ListCard.js]: I removed the height with vh :)

File: client/src/components/ListCard.js (hunk 6 above)
Comment (SebastianStupp) [reply client/src/components/ListCard.js]: I changed the names 👍

File: client/src/components/ListCard.js (hunk 5 above)
Comment (SebastianStupp) [reply client/src/components/ListCard.js]: I corrected the variable :) Maybe I will change the example location with the first refactor.

**Pull request merged**